### PR TITLE
Rewrite readme

### PR DESCRIPTION
**Pull request by benthorner**
This is an attempt to make the project easier for someone (like me!) getting started. I've rewritten the README to make it more concise but also expose some of the 'magic' that was lurking in the setup script, which is unusual for all of the rails apps I've seen.

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -20,11 +20,6 @@ chdir APP_ROOT do
   system! 'gem install bundler --conservative'
   system('bundle check') || system!('bundle install')
 
-  puts "\n== Copying sample files =="
```
Comment (andrewgarner) [inline bin/setup]: I'm in favour of keeping this step. The purpose of the `bin/setup` and `bin/update` scripts are to automate the steps needed to bootstrap the application.

Comment (benthorner) [reply bin/setup]: OK. New version hopefully avoids file clutter.

**Comment by benthorner** [comment]
@andrewgarner Could you take another look at this? Hopefully OK now :-)

File: README.md [hunk 2]
```diff
@@ -1,32 +1,39 @@
...
 
-Copy the example environment file and modify the settings to reflect your configuration.
-`cp .example-env .env`
+Most of these can be installed with Homebrew on a Mac.
 
-> For development you will need a local install of [Publish Data](https://github.com/alphagov/datagovuk_publish/) to populate the databases
+## Getting Started
 
-## Run the app
+First create a `.env` file with the following contents.
```
Comment (andrewgarner) [inline README.md]: We can simply say "run `./bin/setup`". We don't want to maintain an automated way of setting up the application and then duplicate those steps again in the README with the risk they might diverge.

File: README.md [hunk 3]
```diff
@@ -1,32 +1,39 @@
...
-`cat .ruby-version`
+You will need to install the following for development.
 
-Set up the app's dependencies:
-`cd find_data_beta`
-`.bin/setup`
+  * [rbenv](https://github.com/rbenv/rbenv) or similar to manage ruby versions
+  * [bundler](https://rubygems.org/gems/bundler) to manage gems
+  * [elasticsearch](https://www.elastic.co/) database
+  * [postgresql](https://www.postgresql.org/)database
```
Comment (andrewgarner) [inline README.md]: We're missing a space.

File: README.md [hunk 4]
```diff
@@ -1,32 +1,39 @@
...
-Ensure you have the correct version of Ruby installed. You can find the expected version with:
-`cat .ruby-version`
+You will need to install the following for development.
 
-Set up the app's dependencies:
-`cd find_data_beta`
-`.bin/setup`
+  * [rbenv](https://github.com/rbenv/rbenv) or similar to manage ruby versions
+  * [bundler](https://rubygems.org/gems/bundler) to manage gems
+  * [elasticsearch](https://www.elastic.co/) database
```
Comment (andrewgarner) [inline README.md]: Elasticsearch is a search engine not a database.

File: README.md (hunk 2 above)
Comment (benthorner) [reply README.md]: OK. I'm going to remove the final step from bin/setup as it assumes you want to start the app, which technically isn't part of the setup e.g. I might want to run rails c.

**Pull request merged**